partition/bootloader.go

```diff
@@ -43,27 +42,22 @@ var (
 	ErrBootloader = errors.New("cannot determine bootloader")
 )
 
-type bootloaderName string
-
-type bootLoader interface {
-	// Name of the bootloader
-	Name() bootloaderName
-
+// BootloaderIface provides an interface to interact with the system
+// bootloader
+type BootloaderIface interface {
```

Comment:
This can be just `Bootloader` now that it is public, since the casing doesn't conflict with anything else.